fluentd/scripts/dump_activities.rb

```diff
@@ -2,7 +2,8 @@
 require 'date'
 
 def get_audit_activities
-  datetime_from = DateTime.now - 16.0/(24*60)
+  extract_interval = (ENV['LOG_EXPORT_CONTAINER_EXTRACT_AUDIT_ACTIVITIES_INTERVAL'] || '15').to_i
+  datetime_from = DateTime.now - (extract_interval + 1.0)/(24*60)
   datetime_to = DateTime.now - 1.0/(24*60)
   output = `/home/fluent/sdm audit activities -j -e --from "#{datetime_from.to_s}" --to "#{datetime_to.to_s}"`
```

Comment:
Remove absolute path from here. Imagine we run this in a machine, not a container. SDM should be available in the PATH